receiver/k8sobjectsreceiver/factory.go

```diff
@@ -30,6 +30,10 @@ func createDefaultConfig() component.Config {
 		APIConfig: k8sconfig.APIConfig{
 			AuthType: k8sconfig.AuthTypeServiceAccount,
 		},
+		LeaderElection: k8sconfig.LeaderElectionConfig{
+			Enabled:          false,
+			LeaderElectionID: "e3385b9a.leader-election.opentelemetry-collector",
```

Comment:
What's where did `e3385b9a` come from? Ideally, this should be the pod's name. Can't this be set using the in-cluster client?